README.md

```diff
@@ -195,6 +199,7 @@ Multipart/form-data (files) support is based on [`express/multer`](https://githu
 
 ### Fastify support
 
+Fastify support requires `fastify-url-data` plugin to be registered in your application.
```

Comment:
i think that there are two options to tackle this.

1. declare peer dependency
2. implement it's logic

personally, i prefer the latter

Reply:
I checked, and it's not a lot of logic, so I'll just move it over

Reply:
However, under-the-hood it uses uri-js, and that one `is` a lot of code. So I think I will add uri-js as an optional peer dependency, throw an error when creating plugin while not having it as a dependency, and remove need of fastify-url-data